code/modules/cargo/bounties/science.dm

```diff
@@ -132,28 +132,28 @@
 /datum/bounty/item/science/anomaly_core
 	name = "Anomaly Core"
 	description = "A new theory has begun that each sector of space has different anomalies, this all started when a local station tried to make a fire based reactive suit and failed making a stealth version, please send us a core so we may study it more."
-	reward = 2500
+	reward = 5500 //Skyrat edit
 	required_count = 1
 	wanted_types = list(/obj/item/assembly/signaler/anomaly)
 
 /datum/bounty/item/science/anomaly_neutralizer
 	name = "Anomaly Neutralizers"
 	description = "An idea for a long time was to use an unstable Supermatter Shard to help create the breeding grounds for an unstable part of space to harvest any anomalies we want. It worked a little too well and now we're out of anomaly neutralizers, please send us a baker's dozen."
-	reward = 2500
+	reward = 5000 //Skyrat edit
 	required_count = 13
 	wanted_types = list(/obj/item/anomaly_neutralizer)
 
 /datum/bounty/item/science/integrated_circuit_printer
 	name = "Integrated Circuit Printer"
 	description = "Due to a paperwork error, a newly made integrated circuit manufacturer line is missing three of its printers needed to operate. Until the paper work is corrected we are outsourcing this problem, so please send us three integrated circuit printers."
-	reward = 2000
+	reward = 2500 //Skyrat edit
 	required_count = 3
 	wanted_types = list(/obj/item/integrated_circuit_printer)
 
 /datum/bounty/item/science/integrated_circuit_disks
 	name = "Integrated Circuit Printer Upgrade Disks"
 	description = "HR has requested ten more integrated circuit printer upgrade disks, please send them to CC as soon as possible."
-	reward = 2000
+	reward = 2000 
```

Comment:
have a //skyrat edit here

Reply:
YOU'RE TAKING THE FUCKING PISS IT'S A SINGULAR SPACE

Reply:
have a //skyrat edit here